discord_bot/discord_bot.py

```diff
@@ -123,7 +122,9 @@ def get_gateway(self):
                     self.last_seq = message["s"]
 
                 if message["op"] == Opcodes.HELLO:
-                    asyncio.ensure_future(self.handshake(message))
+                    self.heartbeat_interval_ms = message["d"]["heartbeat_interval"]
+                    asyncio.ensure_future(self.heartbeat())
+                    asyncio.ensure_future(self.handshake())
```

Comment:
I prefer keeping setting heartbeat interval and starting the heartbeat in the `handshake()` method, but this isn't wrong. Since this opcode checking is gonna make this function huge, whenever there's a chance to move code out of here I would choose to prefer it.